src/stakeout.js

```diff
@@ -24,7 +24,9 @@ function notifyStakeouts(update, rootCursor) {
 
   if (path === '') {
     // This is a "set" call on the root cursor, so notify based on the properties of the new value
-    each(update.value, (v, key) => notifyStakeouts({ path: key, value: v }, rootCursor));
+    each(update.value, (v, key) => {
+      notifyStakeouts({ path: [key], value: v, action: update.action }, rootCursor);
+    });
```

Comment:
This now conforms with the `StateUpdate` type. The path is always an array and the `action` property was missing.